testPI2/integration/helpers/transferTestHelper.js

```diff
@@ -0,0 +1,193 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+ * Valentin Genev <valentin.genev@modusbox.com>
+ * Nikolay Anastasov <nikolay.anastasov@modusbox.com>
+ --------------
+ ******/
+
+'use strict'
+
+const TransferPreparationModule = require('./transfer')
+const TransferStatePreparationHelper = require('./transferState')
+const StateChangeModel = require('../../../src/domain/transfer/models/transferStateChanges')
+const ExtensionModel = require('../../../src/models/extensions')
+const IlpModel = require('../../../src/domain/transfer/models/ilp-model')
+const TransferModel = require('../../../src/domain/transfer/models/transfers-read-model')
+
+// const preparedData = {
```

Comment:
is this commented out code needed for anything?

Reply:
I'm not sure as probably it also was introduced by **Merge vgenev feature/#248**. Valentin?

Reply:
no we will merge yours and they will need to pull from developPI2 into their branches and merge back in